test/aircraft/Pilot/Pilot.spec.js

```diff
@@ -217,46 +225,90 @@ ava('.replaceFlightPlanWithNewRoute() returns a success message when finished su
     const pilot = buildPilotWithComplexRoute();
     const result = pilot.applyPartialRouteAmendment(invalidRouteString);
 
-    t.true(_isEqual(result, expectedResult));
+    t.deepEqual(result, expectedResult);
 });
 
-ava.skip('.applyPartialRouteAmendment() returns an error with passed a routeString without a shared waypoint', (t) => {
+ava('.applyPartialRouteAmendment() returns an error with passed a routeString without a shared waypoint', (t) => {
     const expectedResult = [false, 'requested route of "HITME..HOLDM" is invalid, it must contain a Waypoint in the current route'];
     const pilot = buildPilotWithComplexRoute();
     const result = pilot.applyPartialRouteAmendment('HITME..HOLDM');
 
-    t.true(_isEqual(result, expectedResult));
+    t.deepEqual(result, expectedResult);
 });
 
-ava.skip('.applyPartialRouteAmendment() returns to the correct flightPhase after a hold', (t) => {
+ava('.applyPartialRouteAmendment() returns to the correct flightPhase after a hold', (t) => {
     const pilot = buildPilotWithComplexRoute();
-    pilot._fms.setFlightPhase('HOLD');
-
+    pilot._fms.setFlightPhase(FLIGHT_PHASE.HOLD);
     pilot.applyPartialRouteAmendment(amendRouteString);
 
-    t.true(pilot._fms.currentPhase === 'CRUISE');
+    // workaround: t.true(pilot._fms.currentPhase) causes out of memory crash
+    const currentPhase = pilot._fms.currentPhase;
+    t.true(currentPhase === FLIGHT_PHASE.CRUISE);
+});
+
+ava('.applyPartialRouteAmendment() returns a success message when complete', (t) => {
+    const expectedResult = [
+        true,
+        {
+            log: 'rerouting to: HITME..HOLDM..BIKKR..DAG.KEPEC3.KLAS',
+            say: 'rerouting as requested'
+        }
+    ];
+    const pilot = buildPilotWithComplexRoute();
+    const result = pilot.applyPartialRouteAmendment(amendRouteString);
+
+    t.deepEqual(result, expectedResult);
 });
 
-ava.skip('.applyPartialRouteAmendment() returns a success message when complete', (t) => {
+ava('.applyPartialRouteAmendment() calls #_fms.applyPartialRouteAmendment()', (t) => {
+    const pilot = buildPilotWithComplexRoute();
+    const fmsApplyPartialRouteAmendmentSpy = sinon.spy(pilot._fms, 'applyPartialRouteAmendment');
     const expectedResult = [
         true,
         {
             log: 'rerouting to: HITME..HOLDM..BIKKR..DAG.KEPEC3.KLAS',
             say: 'rerouting as requested'
         }
     ];
+    const result = pilot.applyPartialRouteAmendment(amendRouteString);
+
+    t.true(fmsApplyPartialRouteAmendmentSpy.calledWithExactly(amendRouteString));
+    t.deepEqual(result, expectedResult);
+});
+
+ava('.applyPartialRouteAmendment() does not grant departure clearance when the route amendment fails', (t) => {
+    const pilot = buildPilotWithComplexRoute();
+
+    t.false(pilot.hasDepartureClearance);
+
+    const expectedResult = [false, `requested route of "${invalidAmendRouteString}" is invalid`];
```

Comment:
@erikquinn  I changed the readback to fix the test. Orginally, it was `routes do not have continuity!`. If you are not happy with the new message, let me know.